### PR TITLE
Add nested choice container by hand decoding, fix disabled chord test

### PR DESCRIPTION
**Pull request by bwetherfield**

### CODE REVIEW

File: Tests/MusicXMLTests/Complex Types/NoteTests.swift [hunk 1]
```diff
@@ -104,7 +104,7 @@ class NoteTests: XCTestCase {
     }
 
     #warning("FIXME: #68 Chord not decoding properly")
```
Comment (jsbean) [inline Tests/MusicXMLTests/Complex Types/NoteTests.swift]: Do we get to kill this `#warning` ?!

File: Sources/MusicXML/Complex Types/Note.swift [hunk 2]
```diff
@@ -142,8 +146,22 @@ extension Note: Codable {
         let ties = try container.decodeIfPresent([Tie].self, forKey: .tie).map(Ties.init)
 
         // Decode pitch / unpitched / rest
-        let pitchUnpitchedRestContainer = try decoder.singleValueContainer()
-        let pitchUnpitchedOrRest = try pitchUnpitchedRestContainer.decode(PitchUnpitchedOrRest.self)
+        let pitchUnpitchedOrRest: PitchUnpitchedOrRest
+        do {
```
Comment (jsbean) [inline Sources/MusicXML/Complex Types/Note.swift]: Can we try the `if container.contains(.pitch)` method that @DJBen has bestowed upon us?

Comment (bwetherfield) [reply Sources/MusicXML/Complex Types/Note.swift]: ah, sure!

File: Tests/MusicXMLTests/Complex Types/NoteTests.swift (hunk 1 above)
Comment (bwetherfield) [reply Tests/MusicXMLTests/Complex Types/NoteTests.swift]: Yes

**Comment by jsbean** [comment]
40 more LilyPond tests parsing without throwing up!

**Comment by jsbean** [comment]
🎆

**Pull request merged**